virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -96,15 +96,15 @@ See xref:../../architecture/architecture-rhcos.adoc#rhcos-about_architecture-rhc
 
 * Supported by {product-title}. See xref:../../scalability_and_performance/optimization/optimizing-storage.adoc#_optimizing-storage[Optimizing storage].
 
-* {rh-storage-first}: You must create a dedicated storage class for Windows virtual machine disks. See link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] for details.
+* You must create a default {VirtProductName} or {product-title} storage class. If both {VirtProductName} and {product-title} default storage classes exist, the {VirtProductName} class takes precedence when creating VM disks.
 
 [NOTE]
 ====
-You must specify a default storage class for the cluster. See xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]. If the default storage class provisioner supports the `ReadWriteMany` (RWX) xref:../../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[access mode], use the RWX mode for the associated persistent volumes for optimal performance.
-
-If the storage provisioner supports snapshots, there must be a xref:../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#volume-snapshot-crds[`VolumeSnapshotClass`] object associated with the default storage class.
+To mark a storage class as the default for virtualization workloads, use the `storageclass.cdi.kubevirt.io/is-default-class` annotation.
```

Comment:
Sorry I misled you, this is the annotation from the design doc, but in the implementation, they changed it. 
The correct annotation is:
storageclass.kubevirt.io/is-default-virt-class: "true"